blocks/search-results/search-results.js

```diff
@@ -0,0 +1,128 @@
+import { createCardBlock } from '../card/card.js';
+
+let done = true;
+
+export default async function decorate(block) {
+  block.innerHTML = '';
+  const searchTerm = new URLSearchParams(window.location.search).get('q');
+  block.classList.add('card-container', 'three-columns');
+  if (searchTerm) {
+    const elementHeading = block.parentNode.parentNode.parentNode.querySelector('.section.search-page-heading');
+    const textNode = document.createElement('h1');
+    textNode.textContent = searchTerm;
+    elementHeading.append(textNode);
+    const mainDiv = document.createElement('div');
+    mainDiv.innerHTML = '<div class="results-loading-spinner"><div>';
+    const resultsDiv = document.createElement('div');
+    resultsDiv.classList.add('results-div');
+    mainDiv.append(resultsDiv);
+    block.append(mainDiv);
+    const tokenizedSearchWords = searchItems(searchTerm);
+    loadResults(tokenizedSearchWords, resultsDiv);
+  }
+}
+
+function searchItems(searchTerm) {
+  const tokenizedSearchWords = searchTerm.split(' ');
+  if (tokenizedSearchWords.length > 1) tokenizedSearchWords.unshift(searchTerm);
+  return tokenizedSearchWords;
+}
+
+async function loadResults(tokenizedSearchWords, resultsDiv) {
+  const allData = await fetch(`${window.location.origin}/articles.json?sheet=full`);
+  console.log(allData);
+  const jsonData = await allData.json();
+  const resultData = filterMatches(tokenizedSearchWords, jsonData);
```

Comment:
I would call this matches, so it's clear this is not the original list. 
```suggestion
  const matches = filterMatches(tokenizedSearchWords, jsonData);
```

Reply:
Fixed